pkgs/by-name/ty/typesetter/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  rustPlatform,
+
+  # nativeBuildInputs
+  cargo,
+  desktop-file-utils,
+  gettext,
+  meson,
+  ninja,
+  pkg-config,
+  python3,
+  rustc,
+  wrapGAppsHook4,
+
+  # buildInputs
+  gdk-pixbuf,
+  graphene,
+  gtk4,
+  gtksourceview5,
+  libadwaita,
+  libspelling,
+  openssl,
+  pango,
+
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "typesetter";
+  version = "0.5.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "haydn";
+    repo = "typesetter";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-TYSmkjYvRUlKxilKr7WlUYQ2U/PsSuI/b8HOnUreXYk=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-lizLARb0wL3hHRl3kbxrlWyt0rB1BUuoZbaVIUeFMXU=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cargo
+    desktop-file-utils
+    gettext # msgfmt
+    meson
+    ninja
+    pkg-config
+    python3
+    rustPlatform.cargoSetupHook
+    rustc
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    graphene
+    gtk4
+    gtksourceview5
+    libadwaita
+    libspelling
+    openssl
+    pango
+  ];
+
+  env.OPENSSL_NO_VENDOR = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "A minimalist, local-first Typst editor";
```

Comment:
Not start with the definite or an indefinite article.

Reply:
Good catch. Removed.